cms_app/cms_app/settings.py

```diff
@@ -0,0 +1,173 @@
+"""
```

Comment:
I don't think we want cms_app level-settings, we want to use the same settings we have in our config folder